runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -880,6 +894,109 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 		addLoot(event, -1, LootRecordType.EVENT, metadata, items);
 	}
 
+	private boolean handleWildernessLootKeys()
+	{
+		boolean recordedLoot = false;
+
+		for (int index = 0; index < WILDERNESS_LOOT_KEY_CONTAINERS.length; index++)
+		{
+			Collection<ItemStack> items = toItemStacks(client.getItemContainer(WILDERNESS_LOOT_KEY_CONTAINERS[index]));
+			if (items.isEmpty())
+			{
+				continue;
+			}
+
+			String value = getWildernessLootKeyValue(index);
+			addLoot("Loot Chest", -1, LootRecordType.EVENT, Strings.isNullOrEmpty(value) ? null : value, items);
+			recordedLoot = true;
+		}
+
+		if (!recordedLoot)
+		{
+			Collection<ItemStack> legacyItems = toItemStacks(client.getItemContainer(InventoryID.LOOT_INV_ACCESS));
+			if (!legacyItems.isEmpty())
+			{
+				addLoot("Loot Chest", -1, LootRecordType.EVENT, null, legacyItems);
+				return true;
+			}
+		}
+
+		return recordedLoot;
+	}
+
+	private Collection<ItemStack> toItemStacks(@Nullable ItemContainer container)
```

Comment:
Make the few other places which have this container stream/map/collect code then call this instead.